tests/phpstan-autoload.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+// moc the windows-only COM class so that the autoloader understands it
```

Comment:
my fault, should be "phpstan"
```suggestion
// moc the windows-only COM class so that phpstan understands it
```